compiler/src/dmd/s2ir.d

```diff
@@ -152,13 +152,8 @@ private extern (C++) class S2irVisitor : Visitor
         bcond.appendSucc(blx.curblock);
         if (s.ifbody)
         {
-            bool ctfe = s.isIfCtfeBlock();         // __ctfe is always false at runtime
-            const isFalse = ctfe;
-
-            irs.falseBlock += isFalse;
-            Statement_toIR(s.ifbody, irs, &mystate);
-            if (isFalse && irs.falseBlock)
-                --irs.falseBlock;
+            if (!s.isIfCtfeBlock())         // __ctfe is always false at runtime
+                Statement_toIR(s.ifbody, irs, &mystate);
```

Comment:
Isn't the inverse needed for the else block when `if (!__ctfe) `?

Reply:
Yes, if we really wanted to support that. I'm on the fence about it. There are other __ctfe constructs, too. I figure we make this solid first.